### PR TITLE
DOCS-18192: Add Confluent Cloud cluster to RBAC role binding examples

### PR DESCRIPTION
**Pull request by tikimims**
Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok
   * no: DO NOT MERGE until the required features are live in prod  

What
----
- Customer pointed out that ccloud cluster is missing from RBAC role binding example.
- https://confluentinc.atlassian.net/browse/DOCS-18192

References
----------
- https://confluent.zendesk.com/agent/tickets/123340

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

File: internal/cmd/iam/command_rbac_role_binding_create.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func (c *roleBindingCommand) newCreateCommand() *cobra.Command {
 			},
 			examples.Example{
 				Text: `Grant the role "ResourceOwner" to the principal "User:u-123456", in the environment "env-12345" for the Kafka cluster "lkc-123456" on the resource "Topic:my-topic":`,
-				Code: "confluent iam rbac role-binding create --principal User:u-123456 --role ResourceOwner --resource Topic:my-topic --environment env-12345 --kafka-cluster lkc-123456",
+				Code: "confluent iam rbac role-binding create --principal User:u-123456 --role ResourceOwner --resource Topic:my-topic --environment env-12345 --cloud-cluster lkc-123456 --kafka-cluster lkc-123456",
```
Comment (brianstrauch) [inline internal/cmd/iam/command_rbac_role_binding_create.go:29]: This is technically not needed. It's inferred from the `--kafka-cluster` flag.

Comment (tikimims) [reply internal/cmd/iam/command_rbac_role_binding_create.go:29]: @brianstrauch Thanks for taking a look @brianstrauch. A customer and Support rep mentioned stated that the examples currently don't. Can you take a look at the customer/support discussion in these two tickets: 

- https://confluentinc.atlassian.net/browse/DOCS-18192
- Support's last comment here: https://confluent.zendesk.com/agent/tickets/123340

Comment (brianstrauch) [reply internal/cmd/iam/command_rbac_role_binding_create.go:29]: Responded in the DOCS ticket. The customer might be running a very old version of the CLI, and they're encountering a separate issue.

Comment (tikimims) [reply internal/cmd/iam/command_rbac_role_binding_create.go:29]: Okay cool. Thanks @brianstrauch. 👍

**Pull request closed without merging**